### ISSUE TITLE
no flic.lua whenth 'main.lua -dataset flic -expID test-run`

### ISSUE THREAD

**Issue by desenzhou** [anewell/pose-hg-train#12]
I execute the command 'th main.lua -dataset flic -expID test-run', but your code does not contain src/dataset/flic.lua,  and after I changed mpii.lua to flic.lua, it shows cannot find data/flic/annot.h5

**Comment by anewell** [comment]
Yeah, sorry about that! In the latest updates to the code infrastructure support for FLIC was temporarily dropped. If you want to test the code on FLIC the commit from a couple months ago still works well, and hopefully I will set up proper support for FLIC in the new code soon.
